packages/host/tests/integration/components/operator-mode-test.gts

```diff
@@ -1070,6 +1077,88 @@ module('Integration | operator-mode', function (hooks) {
       assert.dom('[data-test-message-idx="0"]').doesNotContainText('_love_');
     });
 
+    test('displays message slightly muted when it is being sent', async function (assert) {
+      await setCardInOperatorModeState(`${testRealmURL}Person/fadhlan`);
+      await renderComponent(
+        class TestDriver extends GlimmerComponent {
+          <template>
+            <OperatorMode @onClose={{noop}} />
+            <CardPrerender />
+          </template>
+        },
+      );
+
+      let sendMessageDeffered = new Deferred<void>();
```

Comment:
spelling typos in this test, `sendMessageDeffered`  should be `sendMessageDeferred`